Tribler/Core/Utilities/tracker_utils.py

```diff
@@ -102,10 +102,7 @@ def parse_tracker_url(tracker_url):
     # get port number if exists, otherwise, use HTTP default 80
     if hostname_part.find(u':') != -1:
         hostname, port = hostname_part.split(u':', 1)
-        try:
-            port = int(port)
-        except:
```

Comment:
I discussed this with @whirm. It first raised an Value error, which we would reraise. Might aswell let it throw then.